docs/CHANGELOG.asciidoc

```diff
@@ -37,6 +38,266 @@ This section summarizes the changes in each release.
+* Adds configurable file size to file data visualizer {pull}62752[#62752]
+* Lists global calendars on the job management page {pull}63124[#63124]
+* Improves parsing of large uploaded files {pull}62970[#62970]
+* Changes file data visualizer max upload setting to string {pull}63502[#63502]
+* Adds {kib} setting for file data visualizer max file size {pull}64427[#64427]
+* Enables mml estimation in data recognizer module setup {pull}64900[#64900]
+* Updates memory estimate after adding exclude fields {pull}62850[#62850]
+* Replaces table with data grid {pull}63650[#63650] and {pull}63757[#63757]
+* Fixes feature importance {pull}61761[#61761]
+* Progress including step {pull}65305[#65305]
+Management::
+* Adds Ingest Node Pipelines UI {pull}62321[#62321]
+* Adds Transforms single column wizard {pull}64436[#64436]
+* Edits transform flyout {pull}65033[#65033]
+* Adds docs_per_second to transform edit form {pull}65365[#65365]
+Maps::
+* Separates layer wizards for Clusters and heatmap {pull}60870[#60870]
+* Allows updating requestType for ESGeoGridSource {pull}62365[#62365]
+* Adds date-fields to metrics selection {pull}62629[#62629]
+* Shows create filter button for top-term tooltip property {pull}62461[#62461]
+* Turns on blended layer for geojson upload {pull}63200[#63200]
+* Updates geospatial filters to use geo_shape query for geo_point fields {pull}62966[#62966]
+* Min and max zoom map settings {pull}63714[#63714]
+* Shows spatial filters on map to provide context when for active filters {pull}63406[#63406]
+* Fits geo_shape to bounds {pull}64303[#64303]
+* Disables edit layer button when flyout is open for add layer or map settings {pull}64230[#64230]
+* Do not display EMS or {kib} layer wizards when not configured {pull}64554[#64554]
+* Initial location map settings {pull}64336[#64336]
+* Adds categorical palettes with 20 and 30 categories {pull}64701[#64701]
+* Adds oil rig icon from @elastic/maki {pull}64364[#64364]
+* Highlights selected layer in TOC {pull}61510[#61510]
+* Adds global fit to data {pull}64702[#64702]
+Metrics::
+* View log in context modal {pull}62198[#62198]
+* Refreshes the design for Inventory View, Episode 1 {pull}64026[#64026]
+* Adds Charts to Alert Conditions {pull}64384[#64384]
+* Adds 99th and 95th percentiles to Metric Explorer {pull}64699[#64699]
+* Adds category data quality warning based on ML job stats {pull}60551[#60551]
+* Adds dataset filter to ML module setup screen {pull}64470[#64470]
+* Alerting {pull}62806[#62806]
+* Enhances Alerts management page {pull}64654[#64654]
+* Adds inventory metric threshold alerts {pull}64292[#64292]
+* Adds outside range comparator {pull}63993[#63993]
+Platform::
+* Uses brotli compression for some KP assets {pull}64367[#64367]
+Reporting::
+* Adds a new config flag to encode with BOM for our CSVs {pull}63006[#63006]
+* Adds config flag to escape formula CSV values {pull}63645[#63645]
+Security::
+* Moves jira to a gold license {pull}67178[#67178]
```

Comment:
This belongs under the SIEM heading, rather than the Security heading

Reply:
@legrego the Release Notes is currently broken, but I'll add in the Security team PRs and let you know when they're ready for review.